### PR TITLE
Adding project_path to run.py

### PR DESCRIPTION
**Pull request by mwiewior**

### CODE REVIEW

File: kedro_mlflow/io/mlflow_dataset.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def _save(self, data: Any):
                 # it will throw exception that file under path already exist.
                 local_path = self._get_save_path()
 
+                print(f"Saving dataset local_path:{local_path}, artifact_path:{self.artifact_path}")
```
Comment (kaemo) [inline kedro_mlflow/io/mlflow_dataset.py:40]: Print defintely shouldn't be used here. If anything, it should be logged using a logger, but I have doubts whether that's needed at all. Kedro is not logging save paths and I think we shouldn't either.

@Galileo-Galilei what do you think?

**Review by kaemo — CHANGES_REQUESTED** [review]
Please add an entry to the CHANGELOG.md (see CONTRIBUTING.md for contribution guidelines). It would be great if you described why those changes are introduced - have you encountered any bug?

**Comment by Galileo-Galilei** [comment]
Sorry for the late reply, and thank you very much for contributing. 

I don't think we should merge this one, but I am opened to discussion (to be honest, you will need pretty strong arguments to change my mind :-) but you can give a try and I'll accept to go with the majority if you all stand against my arguments). See the reasons why in #54 and #29. However, I have in mind that integration with airflow is a big deal and I am going to dive into it when I'll have more time.

File: kedro_mlflow/io/mlflow_dataset.py (hunk 1 above)
Comment (Galileo-Galilei) [reply kedro_mlflow/io/mlflow_dataset.py:40]: Yes, I totally agree.

**Comment by Galileo-Galilei** [comment]
Hello, this argument is deprecated when merging #58 so I close the PR. I still some questions however that I will put in # 54 to keep track more easily. Tells us if it fits your need!

**Pull request closed without merging**